Source/WebCore/css/CSSGroupingRule.cpp

```diff
@@ -125,7 +125,8 @@ void CSSGroupingRule::appendCSSTextForItemsInternal(StringBuilder& builder, Stri
         return;
     }
 
-    builder.append(static_cast<StringView>(rules), "\n}");
+    builder.append(rules);
+    builder.append("\n}"_s);
```

Comment:
Sadly this will be slightly less efficient than the uglier way where we cast to `StringView`. The elegant fix would be to deal with the `FIXME` in `StringBuilder.h` and make it work with `StringConcatenate.h`. I guess this is fine for now.

Reply:
Why is it less efficient? `StringBuilder::append(const StringBuilder&)` seems to call the StringView overload anyway. However, it does have a fast path for the case where we're appending to an empty StringBuilder.

Reply:
Two calls to append may grow the `StringBuilder` twice. A single call computes the combined length and grows it at most once.

Reply:
Oh I see. I'll bring back the single append() call.